data/es/conferences.yml

```diff
@@ -1,5 +1,13 @@
 
 items:
+  -
+    name: Jakarta One Livestream - Russian
+    location: Location, Location
+    date: March 31, 2021
+    end_date: 2021-04-01T00:00:00-00:00
+    url: /2021/russia
+    logo_white: /images/jakartaone_livestream_wht_full-06.svg
+    logo_color: /images/jakarta_one_logo.png
   -
     name: JakartaOne 2020
     location: Location, Location
```

Comment:
Though the event is in Russian, I think for other language's page I need to make the homepage consistent. So I'm adding this to all the lang home page.